CONTRIBUTING.md

```diff
@@ -209,6 +209,13 @@ The package distributions (Debian and RPM) can be found under:
 The archive distributions (tar and zip) can be found under:
 `./distribution/archives/(tar|zip)/build/distributions/`
 
+Note that this assembles every distributable artifact (e.g. plugins, javadocs) as well as
```

Comment:
`distributable` -> `build`
`(e.g. plugins, javadocs)` -> `(e.g., plugins and Javadocs)`
`javadocs` -> `Javadocs`